circle.yml

```diff
@@ -208,9 +262,25 @@ workflows:
           requires:
             - integration_tests_group_1
             - integration_tests_group_2
-  frontend-build-test:
+
+  frontend-build-and-deploy:
     jobs:
-      - build-frontend
+      - test-frontend
+      - build-frontend:
+          requires:
+            - test-frontend
+      - allow-deployment:
+          type: approval
+          requires:
+            - build-frontend
+      - deploy-frontend-stage:
+          requires:
+            - allow-deployment
+          filters:
+            branches:
+              only:
+                - giovanni/add-frontend-ci-deployment
```

Comment:
This needs to be renamed before merging, and renamed again before merging to `master`